src/test/java/org/reaktivity/nukleus/maven/plugin/internal/generated/ConstrainedMapFWTest.java

```diff
@@ -109,7 +109,7 @@ static void assertAllTestValuesRead(
         int offset)
     {
         List<String> mapItems = new ArrayList<>();
-        flyweight.forEach(kv -> vv ->
+        flyweight.forEach((kv, vv) ->
```

Comment:
Let's use `(k, v)` here instead?
Same feedback for other similarly shaped lambdas.